Automattic/jetpack
# My Jetpack: fix a scrolling issue when changing pages through the Route component

This PR fixes an issue when the user changes the route component, for instance, by clicking on the _Connect your user account_ link, scrolling the window to the top. For this, it creates a new `<ScrollToTop />` component.

Fixes #

#### Changes proposed in this Pull Request:

*

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to My Jetpack
* Disconnect the user to be able to see the _Connect your user account_ link
* Click on the link
* Confirm the window scrolls to the top when loading the Connection page


before | after
------|------
<img width="958" alt="image" src="https://user-images.githubusercontent.com/77539/178758488-df198082-04c6-4f3f-9714-3444c384b903.png"> | <img width="958" alt="image" src="https://user-images.githubusercontent.com/77539/178758626-d2614101-accb-40df-bfac-6e57c2dd9de6.png">
